drivers/adc/ad469x/ad469x.h

```diff
@@ -0,0 +1,236 @@
+/***************************** Include Files **********************************/
+/******************************************************************************/
+#include "spi_engine.h"
+#include "clk_axi_clkgen.h"
+#include "pwm.h"
+#include "gpio.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definitions **********************/
+/******************************************************************************/
+/* AD469x registers */
+#define AD469x_REG_IF_CONFIG_A		0x000
+#define AD469x_REG_IF_CONFIG_B		0x001
+#define AD469x_REG_DEVICE_TYPE		0x003
+#define AD469x_REG_DEVICE_ID_L		0x004
+#define AD469x_REG_DEVICE_ID_H		0x005
+#define AD469x_REG_SCRATCH_PAD		0x00A
+#define AD469x_REG_VENDOR_L		0x00C
+#define AD469x_REG_VENDOR_H		0x00D
+#define AD469x_REG_LOOP_MODE		0x00E
+#define AD469x_REG_IF_CONFIG_C		0x010
+#define AD469x_REG_IF_STATUS		0x011
+#define AD469x_REG_STATUS		0x014
+#define AD469x_REG_ALERT_STATUS1	0x015
+#define AD469x_REG_ALERT_STATUS2	0x016
+#define AD469x_REG_ALERT_STATUS3	0x017
+#define AD469x_REG_ALERT_STATUS4	0x018
+#define AD469x_REG_CLAMP_STATUS1	0x01A
+#define AD469x_REG_CLAMP_STATUS2	0x01B
+#define AD469x_REG_SETUP		0x020
+#define AD469x_REG_REF_CTRL		0x021
+#define AD469x_REG_SEQ_CTRL		0x022
+#define AD469x_REG_AC_CTRL		0x023
+#define AD469x_REG_STD_SEQ_CONFIG	0x024
+#define AD469x_REG_GPIO_CTRL		0x026
+#define AD469x_REG_GP_MODE		0x027
+#define AD469x_REG_GPIO_STATE		0x028
+#define AD469x_REG_TEMP_CTRL		0x029
+
+/* 5-bit SDI Conversion Mode Commands */
+#define AD469x_CMD_REG_CONFIG_MODE		(0x0A << 3)
+#define AD469x_CMD_SEL_TEMP_SNSOR_CH		(0x0F << 3)
+#define AD469x_CMD_CONFIG_CH_SEL(x)		((0x10 | (0x0F & x)) << 3)
+
+/* AD469x_REG_SETUP */
+#define AD469x_SETUP_IF_MODE_MASK		(0x01 << 2)
+#define AD469x_SETUP_IF_MODE_CONV		(0x01 << 2)
+#define AD469x_SETUP_CYC_CTRL_MASK		(0x01 << 1)
+#define AD469x_SETUP_CYC_CTRL_SINGLE(x)		((x & 0x01) << 1)
+
+/* AD469x_REG_GP_MODE */
+#define AD469x_GP_MODE_BUSY_GP_EN_MASK		(0x01 << 1)
+#define AD469x_GP_MODE_BUSY_GP_EN(x)		((x & 0x01) << 1)
+#define AD469x_GP_MODE_BUSY_GP_SEL_MASK		(0x01 << 4)
+#define AD469x_GP_MODE_BUSY_GP_SEL(x)		((x & 0x01) << 4)
+
+/* AD469x_REG_SEQ_CTRL */
+#define AD469x_SEQ_CTRL_STD_SEQ_EN_MASK		(0x01 << 7)
+#define AD469x_SEQ_CTRL_STD_SEQ_EN(x)		((x & 0x01) << 7)
+#define AD469x_SEQ_CTRL_NUM_SLOTS_AS_MASK	(0x7f << 0)
+#define AD469x_SEQ_CTRL_NUM_SLOTS_AS(x)		((x & 0x7f) << 0)
+
+/******************************************************************************/
+/*************************** Types Declarations *******************************/
+/******************************************************************************/
+/**
+ * @enum ad469x_interface_mode
+ * @brief Possible device communication modes
+ */
+enum ad469x_interface_mode {
+	/** Read/Write registers */
+	AD469x_IF_REGISTER_MODE = 0,
```

Comment:
just a matter of preference:
`AD469x_IF_REGISTER_MODE` -> `AD469x_REGISTER_MODE`
`AD469x_IF_CONVERSION_MODE` -> `AD469x_CONVERSION_MODE`